packages/lesspass-pure/package.json

```diff
@@ -26,7 +26,15 @@
   "jest": {
     "testPathIgnorePatterns": [
       "cypress"
-    ]
+    ],
+    "moduleFileExtensions": [
+      "js",
+      "vue"
+    ],
+    "transform": {
+      ".*\\.(vue)$": "vue-jest",
+      "^.+\\.js$": "../../node_modules/babel-jest"
+    }
```

Comment:
Should I move all this to the top level ?

Reply:
It will be ok to keep jest config in this package.json no?